generic-helpers/src/main/java/io/elimu/a2d2/helpers/PasswordDecryptUtil.java

```diff
@@ -31,6 +31,7 @@ private PasswordDecryptUtil() {
 	protected static StringEncryptor stringEncryptor() {
 		String salt = System.getenv("jasypt_encryptor_password");
 		if (salt == null || "".equals(salt.trim())) {
+			LOG.debug("Couldn't find environment variable jasypt_encryptor_password");
```

Comment:
we might want to use LOG.warn here

Reply:
Agree. Please make this change before merging.